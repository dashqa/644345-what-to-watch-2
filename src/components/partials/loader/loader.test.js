import React from 'react';
import renderer from 'react-test-renderer';
import Loader from './loader';

it(`Loader correctly renders after relaunch`, () => {
  const tree = renderer.create(<Loader/>).toJSON();
  expect(tree).toMatchSnapshot();
});
