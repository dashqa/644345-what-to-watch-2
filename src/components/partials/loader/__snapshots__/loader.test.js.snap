// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader correctly renders after relaunch 1`] = `
<div
  style={
    Object {
      "left": "50vw",
      "position": "absolute",
      "top": "50vh",
      "transform": "translate(-50%, -50%)",
    }
  }
>
  <svg
    className="sc-bdVaJa khEZTZ"
    height="250px"
    viewBox="0 0 251 251"
    width="250px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <circle
      className="sc-bwzfXH gScrst"
      cx={125.5}
      cy={125.5}
      fill="none"
      r={115}
      strokeLinecap="round"
      strokeWidth={20}
    />
  </svg>
</div>
`;
